README.md

```diff
@@ -126,6 +127,12 @@ You can find high level and deep dive videos on [videos.traefik.io](https://vide
 
 ## Maintainers
 
+We are strongly promoting a philosophy of openness and sharing, and firmly standing against the elitist closed approach. Being part of the core team should be accessible to anyone who is motivated and want to be part of that journey!
+This [document](docs/content/contributing/maintainers-guidelines.md) describes how to be part of the core team as well as various responsibilities and guidelines for Traefik maintainers.
+You can also find more information on our process to review pull requests and manage issues [in this document](docs/content/contributing/maintainers.md).
+
+## Maintainers
```

Comment:
Not sure why this section is still here 🤔

Reply:
I removed the duplicated header Maintainers. Not sure if we should remove the link to the Maintainers guidelines and to The Maintainers. I left it unchanged for now.